cypress/plugins/index.js

```diff
@@ -1,9 +1,6 @@
 /// <reference types="cypress" />
 /* eslint-env node */
 
-const { spawn } = require('child_process');
```

Comment:
We no longer need these plugins as cypress-rails is handling seed data as well as resetting the database.